tezosqa/1.0/Dockerfile

```
@@ -0,0 +1,39 @@
+FROM python:3.8-slim-buster
+
+ARG SMARTPY_VERSION
+ARG SMARTPY_PATH
+
+ENV SMARTPY_INSTALL_URL https://smartpy.io/releases/${SMARTPY_VERSION}/cli
+
+RUN echo "Install system dependencies for python and pip" && \
+    #echo "path : " && smpat = $(${SMARTPY_PATH}) && mkdir ${smpat} && ls -la ~ && exit 1 &&\
+    pip install -U pip && \
+    echo "Done base install!" && \
+    echo "Install basics Python tools" && \
+    pip install pipenv && \
+    echo "Done!" && \
+    echo "Install crypto libs" && \
+    apt-get update -q -y && apt-get install -q -y libsodium-dev libsecp256k1-dev libgmp-dev gcc git g++ make && \
+    echo "Done!" && \
+    echo "Install SmartPyDev" && \
+    apt-get install -q -y curl && \
+    curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+    apt-get install -y nodejs && \
+    curl -s "${SMARTPY_INSTALL_URL}/install.sh" > install.sh && bash install.sh --prefix ${SMARTPY_PATH} --from ${SMARTPY_INSTALL_URL} --yes && rm install.sh && \
```

You don't need to rm anything if you don't create a file in the first place, check if this works:
```suggestion
     bash <(curl -s "${SMARTPY_INSTALL_URL}/install.sh") --prefix ${SMARTPY_PATH} --from ${SMARTPY_INSTALL_URL} --yes && \
```